zebra-state/src/service.rs

```diff
@@ -389,8 +389,7 @@ impl StateService {
         let full_verifier_utxo_lookahead = max_checkpoint_height
             - HeightDiff::try_from(checkpoint_verify_concurrency_limit)
                 .expect("fits in HeightDiff");
-        let full_verifier_utxo_lookahead =
-            full_verifier_utxo_lookahead.expect("unexpected negative height");
+        let full_verifier_utxo_lookahead = full_verifier_utxo_lookahead.unwrap_or(block::Height(0));
```

Comment:
> Zebra should not panic without checkpoints.

I think this code change is helpful, but the explanation isn't correct.

Zebra must have checkpoints up to `mandatory_checkpoint_height()`. If it doesn't, then it could break consensus rules:
https://github.com/ZcashFoundation/zebra/blob/290ccf2d94e68b80008517614b4f3873e06a93fd/zebra-chain/src/parameters/network.rs#L96-L102

Is there a specific situation or bug that this change fixes?
That might be helpful context to put in the PR description.